db/schema.rb

```diff
@@ -259,6 +259,7 @@
     t.boolean "has_close_relationships"
     t.boolean "has_right_to_work_in_uk"
     t.boolean "has_safeguarding_issue"
+    t.string "type"
```

Comment:
this should eventually be non-nullable (Think Vacancy needs to have its default removed...)

Reply:
Am planning to address this one in a follow up PR once we have no null types left after running the rake task!